### DISCUSSION TITLE
Temporary failure resolving syncthing-apt.svc.edge.scw.cloud

### DISCUSSION THREAD

**Discussion by mmaslen** [tteck/Proxmox#3863] [Help Needed]
Asking this here as I am unable to log an "issue"

getting this error running the syncthing script - this worked perfectly last week.

Now getting errors: 

|The following NEW packages will be installed:
  syncthing
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 10.9 MB of archives.
After this operation, 26.9 MB of additional disk space will be used.
Ign:1 https://syncthing-apt.svc.edge.scw.cloud syncthing/stable amd64 syncthing amd64 1.27.12
Ign:1 https://syncthing-apt.svc.edge.scw.cloud syncthing/stable amd64 syncthing amd64 1.27.12
Ign:1 https://syncthing-apt.svc.edge.scw.cloud syncthing/stable amd64 syncthing amd64 1.27.12
Err:1 https://syncthing-apt.svc.edge.scw.cloud syncthing/stable amd64 syncthing amd64 1.27.12
  Temporary failure resolving 'syncthing-apt.svc.edge.scw.cloud'
E: Failed to fetch https://syncthing-apt.svc.edge.scw.cloud/dists/syncthing/stable/binary-amd64/syncthing_1.27.12_amd64.deb  Temporary failure resolving 'syncthing-apt.svc.edge.scw.cloud'
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?

**Comment by Mellowlynx** [comment]
I just checked the DNS of the domain and got a DNSsec error.
If you have DNSsec on, disable it and try again after 4 hours.
Else, it might be a local DNS error.
